src/components/ForecastSummary.js

```diff
@@ -0,0 +1,40 @@
+import React from "react";
+import WeatherIcon from "react-icons-weather";
+import PropTypes from "prop-types";
+import "../styles/ForecastSummary.css";
+
+function ForecastSummary(props) {
+  const { date, description, icon, onSelect, temperature } = props;
```

Comment:
You can do this in the brackets in the function definition for the component and save yourself a line of code. Like this:
`function ForecastSummary({ date, description, icon, onSelect, temperature }) {`